Neos.Media/Classes/Command/MediaCommandController.php

```diff
@@ -388,6 +410,164 @@ public function renderThumbnailsCommand(int $limit = null, bool $quiet = false)
+                foreach ($presetVariantNames as $variants) {
+                    $outputPresets[] = [$presetIdentifier, $variants];
+                }
+            }
+            $this->outputLine('There is a <b>configuration</b> with following <b>variant presets:</b>');
+            $this->output->outputTable($outputPresets, ['Configured Presets', 'Label of variant']);
+        }
+
+        $databaseVariants = $this->imageVariantRepository->findAllWithOutdatedPresets($currentPresets);
+
+        if (empty($databaseVariants)) {
+            $this->outputLine('There were <b>no outdated variant presets</b> in your database...');
+        } else {
+            $formattedDatabaseVariants = [];
+            foreach ($databaseVariants as $variant) {
+                if (!isset($formattedDatabaseVariants[$variant->getPresetIdentifier()][$variant->getPresetVariantName()])) {
+                    $formattedDatabaseVariants[$variant->getPresetIdentifier()] = [$variant->getPresetVariantName() => 1];
+                } else {
+                    $formattedDatabaseVariants[$variant->getPresetIdentifier()][$variant->getPresetVariantName()]++;
+                }
+            }
+
+            $outputPresets = [];
+            foreach ($formattedDatabaseVariants as $presetIdentifier => $presetVariantNames) {
+                $presetVariantKeys = array_keys($presetVariantNames);
+
+                foreach ($presetVariantKeys as $key) {
+                    $outputPresets[] = [$presetIdentifier, $key, $presetVariantNames[$key]];
+                }
+            }
+            $this->outputLine('There is a <b>configuration</b> with following <b>variant presets:</b>');
+            $this->output->outputTable($outputPresets, ['Found Presets', 'Outdated variant label', 'Times found']);
+
+            $this->outputLine('To delete them, please use the <b>removeUnusedImageVariants</b> command.');
+        }
+
+        die;
+    }
+
+    /**
+     * Tidy up outdated imageVariants with provided identifier and variant name.
+     *
+     * This command iterates over all existing asset variants with outdated preset.
+     * They can be deleted if not used.
+     *
+     * To delete custom crops made in neos backend, set identifier and name to NULL:
+     * `media:removeoutdatedvariants --identifier NULL --variant-name NULL`
+     *
+     * @param string $identifier Identifier of variants to remove.
+     * @param string $variantName Variants with this name will be removed (if exist).
+     * @param bool $quiet If set, only errors and questions will be displayed.
+     * @param bool $assumeYes If set, "yes" is assumed for the "shall I remove ..." dialog.
+     * @param int|null $limit Limit the result of unused assets displayed and removed for this run.
+     *
+     * @throws StopCommandException
+     */
+    public function removeOutdatedVariantsCommand(string $identifier, string $variantName, bool $quiet = false, bool $assumeYes = false, int $limit = null)
```

Comment:
Would it make sense to have a distinct `removeCustomCropVariantsCommand` instead of setting id and name to `NULL`.

Reply:
Yes, I would also split this in a distinct command as `removeoutdatedvariants` implies that a subset of variants is cleaned. Using `--identifier NULL --variant-name NULL` would AFAIK delete *all* custom crops.

That sounds kind of harmful for me.

Additionally we should split this use case in another issue and not add it on top of this changes.

Reply:
Addition: Is the function name still correct? You can now remove variants that are still configured.